### ISSUE TITLE
Can't change the text for no feedback

### ISSUE THREAD

**Issue by csdevteam** [shaileshmamgain5/TwoStageRate#3]
Hello. Please add a functionality for change the text "Bro.. Write Something" for any text. I need change it to another language

**Comment by mishindmitriy** [comment]
Hi, try replace string resource with the same id

**Comment by shaileshmamgain5** [comment]
Yes. Will add this in next version within this week. Thank you for pointing out.

**Comment by mkAndroDev** [comment]
Hello, your library is really nice. Could you fix this problem?

**Comment by shaileshmamgain5** [comment]
Yes I understand it is a problem. But cannot fix it right away as there is too much on my plate for weekdays. Will definitely do it on priority this weekend. 

Meanwhile could you please fork and fix it if possible. Or else please wait for 3-4 days. 

Thank you for pointing out. Cheers!
